packages/flutter_web_plugins/lib/src/plugin_registry.dart

```diff
@@ -0,0 +1,124 @@
+// Copyright 2019 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:async';
```

Comment:
Since this is a new package and fairly constrained in scope, you should shoot for close to 100% test coverage. This will make it much easier for new contributors to add on as needed if there are a lot of existing test cases. (We'd need to execute it with -p chrome, but that should be okay long term)

Reply:
Added some tests. However, I have to skip them until `webOnlySetPluginHandler` lands in the Flutter SDK engine.